### PR TITLE
Refs #20317 - more Rails 5.1 updates

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: test/katello_test_helper.rb [hunk 1]
```diff
@@ -69,7 +69,6 @@ module FixtureTestCase
   included do
     extend ActiveRecord::TestFixtures
 
-    self.use_transactional_fixtures = true
```
Comment (jlsherrill) [inline test/katello_test_helper.rb]: should this be changed to use_transactional_tests ?

Comment (mmoll) [reply test/katello_test_helper.rb]: In Foreman this got only removed, as it's not changing anything with Rails 5.x.

What we have in core:
```
$ git grep use_transaction
test/integration_test_helper.rb:  self.use_transactional_tests = false
test/unit/tasks/seeds_test.rb:  self.use_transactional_tests = false
```

Comment (mmoll) [reply test/katello_test_helper.rb]: OK, found https://blog.bigbinary.com/2016/05/26/rails-5-renamed-transactional-fixtures-to-transactional-tests.html "By default this functionality is enabled. "

**Comment by mmoll** [comment]
💚

**Pull request merged**